static/app/gettingStartedDocs/javascript/angular.tsx

```diff
@@ -158,6 +159,23 @@ export const steps = ({
       },
     ],
   },
+  {
+    language: 'bash',
+    type: StepType.UPLOAD_SOURCE_MAPS,
+    configurations: [
+      {
+        description: tct(
+          'Automatically upload your source maps to enable readable stack traces for Errors. If you prefer to manually set up source maps, please follow [guideLink:this guide].',
+          {
+            guideLink: (
+              <ExternalLink href="https://docs.sentry.io/platforms/javascript/guides/angular/sourcemaps/" />
+            ),
+          }
+        ),
+        code: `npx @sentry/wizard@latest -i sourcemaps`,
+      },
+    ],
+  },
```

Comment:
We'll be adding this to a lot of platforms - should we create a reusable function that will accept guide URL and spit out the rest?

Reply:
good idea 💡